conf/test.config

```diff
@@ -10,20 +10,30 @@
 ----------------------------------------------------------------------------------------
 */
 
+process {
+    withName: 'PURPLE' {
+        ext.args = '-min_purity 1 -max_purity 1 -min_ploidy 2 -max_ploidy 2'
+    }
+}
+
 params {
     config_profile_name        = 'Test profile'
     config_profile_description = 'Minimal test dataset to check pipeline function'
 
+    // NOTE(SW): incompatible with GHA given size of reference data, STAR align requires ~30 GB memory
     // Limit resources so that this can run on GitHub Actions
-    max_cpus   = 2
-    max_memory = '6.GB'
+    max_cpus   = 1
+    max_memory = '30.GB'
     max_time   = '6.h'
 
     // Input data
-    // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input  = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_test_illumina_amplicon.csv'
+    input = 'https://pub-349bcb8decb44bf7acbddf90b270a061.r2.dev/simulated_reads/24.0/samplesheets/fastq_eval.subject_a-subject_b.wgts.tndna_trna.1.csv'
```

Comment:
This is also hosted on Cloudflare R2 along with the simulated read data that it points to. Similar to the iGenomes work, I'd like to tackle this in parallel to the release with you or other community members.

Would you be okay starting that process with me creating an issue to coordinate?

Reply:
Done in:

- https://github.com/nf-core/oncoanalyser/pull/72
- https://github.com/nf-core/test-datasets/pull/1270

However this does not include reference data because of the reasons in https://github.com/nf-core/oncoanalyser/pull/29#discussion_r1683962125.